2022Swerve/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,14 +26,18 @@ public class RobotContainer {
 
   private final Joystick driverJoystick = new Joystick(OIConstants.kDriverControllerPort);
 
+  private final SimulationSubsystem simSub = new SimulationSubsystem(swerveSubsystem);
```

Comment:
Why are we abbreviating subsystem as sub? Would it be more consistent to name the subsystem "simulationSubsystem"

Also, small thing, but we should probably be instantiating all subsystems together.

Reply:
Fixed